app/views/categories/_left_nav.html.erb

```diff
@@ -10,14 +10,16 @@
 <div id="left-col" class="col-md-3 hidden-sm hidden-xs affix affix-top">
   <ul class="list-unstyled border-right">
   <% @categories.with_translations(I18n.locale).each do |category| %>
```

Comment:
You could define a new scope here like `@categories.public.with_trans...` and avoid the `if..end`